<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>730</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>331</width>
      <height>381</height>
     </rect>
    </property>
    <property name="title">
     <string>Gestion Des Salles</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>num_salle</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>56</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>etage</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>170</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>departement</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_num_salle">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>50</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_ajouter">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>190</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>ajouter</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_supprimer">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>320</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>supprimer</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>220</y>
       <width>56</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>juge</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>270</y>
       <width>56</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>suspect</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_suspect">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>270</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_num_salle_supp">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>350</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="le_juge">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>230</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Yassin Ghorbel</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Chedli Ben Larbi</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Dorra Gharbi</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Nermine Romdhan</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Amira Allani</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Med Dhifallah</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Adem Salhi</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Youssef Taamallah</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Mariam Guedhami</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Emna Bouden</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="le_departement">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>170</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>A</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>B</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>C</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>D</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>E</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>F</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>G</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>H</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>I</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>J</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>K</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>L</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>M</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>N</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>O</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>P</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Q</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>R</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>S</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>T</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>U</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>V</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>W</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>X</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>W</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Z</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="le_etage">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>110</y>
       <width>73</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>7</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>9</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>11</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>12</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>0</y>
      <width>701</width>
      <height>431</height>
     </rect>
    </property>
    <property name="title">
     <string>Affichage</string>
    </property>
    <widget class="QTableView" name="tab_salle">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>301</width>
       <height>321</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_modif">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>370</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>modifier</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>0</y>
       <width>301</width>
       <height>331</height>
      </rect>
     </property>
     <property name="title">
      <string>Tri et recherche</string>
     </property>
     <widget class="QTableView" name="tab_recherche">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>21</y>
        <width>301</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pb_rechercher">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>380</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Rechercher</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_rechercher">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>380</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>340</y>
       <width>73</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>etage</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>departement</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>num de salle</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="PDF">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>380</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>PDF</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>-20</y>
      <width>1081</width>
      <height>691</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../court1.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>label_6</zorder>
   <zorder>groupBox</zorder>
   <zorder>groupBox_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
